server/pypi/packages/scipy/meta.yaml

```diff
@@ -2,11 +2,15 @@ package:
   name: scipy
   version: "1.8.1"
 
+build:
+  number: 1
+  script_env:
+    - SCIPY_USE_PYTHRAN=0
+
 requirements:
   build:
     - fortran
-    - cython 0.29.36        
```

Comment:
build-wheel now automatically installs `build-system` requirements from pyproject.toml, so most of the Cython requirements in meta.yaml are no longer needed.